Update supported Kubernetes versions in README (#271)
scalar-labs/helm-charts: README.md

## Supported Kubernetes versions

> * 1.30.x, 1.29.x, 1.28.x, 1.27.x, 1.26.x

Reviewer: BTW, can we add a note about how the versions are decided?
The supported versions are the intersection of the supported versions of AKS and EKS at the moment?

Author: > BTW, can we add a note about how the versions are decided?

Ah, I see. I think we can add it. I will add it in this PR.

> The supported versions are the intersection of the supported versions of AKS and EKS at the moment?

Yes. Basically, at this time, I decid on our supported versions based on supported versions of EKS and AKS. (Recently, I check GKE and OKE just in case.)

Author: @feeblefakie  
I added a description in 3d2994bcfc9ecfb215e40442ad6b54d8dde315e0.

@josh-wong 
We added the new sentence in the README. Could you please take a look at this new sentence? 🙇‍♀️

## Supported Kubernetes versions

* 1.30.x, 1.29.x, 1.28.x, 1.27.x, 1.26.x

> We decide on our supported version based on the supported versions in [Amazon Elastic Kubernetes Service](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) and [Azure Kubernetes Service](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions). However, we do not consider LTS versions in each managed Kubernetes service.

Reviewer 2: ```suggestion
We decide which versions to support based on the supported versions in [Amazon Elastic Kubernetes Service](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) and [Azure Kubernetes Service](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions). However, we do not consider the LTS versions in each managed Kubernetes service.
```
